src/bindings/js/node/include/napi_arg.hpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <napi.h>
+
+#include "openvino/runtime/core.hpp"
+
+namespace NapiArg {
+const char* get_type_name(napi_valuetype type);
+
+class Validator {
+public:
+    typedef std::function<void(const std::string& key, const Napi::Value&)> ValidatorType;
+
+    const bool validate(const Napi::CallbackInfo& info, std::vector<std::string>& error_messages);
+    const bool validate(const Napi::CallbackInfo& info);
```

Comment:
Did you mean const method?
```suggestion
    bool validate(const Napi::CallbackInfo& info, std::vector<std::string>& error_messages) const;
    bool validate(const Napi::CallbackInfo& info) const;
```